src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -817,17 +817,24 @@ def __try_add_error(error_list, detail):
         error_list.insert(0, detail)
         return True
 
-    def __set_errors_json(self, error_count_by_operation, errors_by_operation):
+    def __set_errors_json(self, error_count_by_operation, errors_by_operation, is_status_truncated=False):
         """ Compose the error object json to be added in 'errors' in given operation's summary """
+        code = Constants.PatchOperationTopLevelErrorCode.SUCCESS if error_count_by_operation == 0 else Constants.PatchOperationTopLevelErrorCode.ERROR
+
         if error_count_by_operation == 1 and errors_by_operation[0]['code'] == Constants.PatchOperationErrorCodes.INFORMATIONAL:    # special-casing for single informational messages
             message = errors_by_operation[0]['message']
             errors_by_operation = []
-            error_count_by_operation = 0
         else:
+            # Update msg error code to warning for truncation
+            if is_status_truncated:
+                error_count_by_operation += 1  # add 1 because of truncation creates a new error detail object
+                code = Constants.PatchOperationTopLevelErrorCode.WARNING if code != Constants.PatchOperationTopLevelErrorCode.ERROR else Constants.PatchOperationTopLevelErrorCode.ERROR
```

Comment:
Test this on a machine where the results go into CRP API / ARG.

Reply:
the log from condition check has this logic, so far no error from it

done